source/OutfitterPanel.cpp

```diff
@@ -168,6 +168,10 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point)
 
 	const Font &font = FontSet::Get(14);
 	const Color &bright = *GameData::Colors().Get("bright");
+	const Color &highlight = *GameData::Colors().Get("outfitter difference highlight");
+	// Highlight outfit differences only if several ships of the same model are selected:
```

Comment:
This comment should probably be where the logic is actually carried out, and it should also end with a full stop.

Reply:
Much better, thanks!